app/views/lists/show.html.erb

```diff
@@ -10,5 +10,26 @@
   <%= @list.description %>
 </p>
 
+<ul>
+  <% @list.papers.each do |paper| %>
+    <li>
+      <%=paper.title%>
+    </li>
+  <%end%>  
+</ul>
+
+<%= form_for(@reference) do |f| %>
+  <div><%= hidden_field_tag :list_id, @list.id %></div>
+
+
+  <div class="field">
+    <%= f.label :paper_id %>
+    <%= f.text_field :paper_id %>
+  </div>
+
+    <%= f.submit "Add Paper", class: "btn btn-primary" %>
```

Comment:
Another style comment, but this should be in line with the `div`. It's a sibling element